pages/models/regulations/governor/GovSteamEU/index.md

```diff
@@ -0,0 +1,443 @@
+
+## Variables
+
+### Inputs
+
+<div id="tbl-inputs">
+
+Table 2: Inputs
+</div>
+
+| name               | type  | unit | IEC name   | description                          |
+| :----------------- | :---- | :--- | :--------- | :----------------------------------- |
+| $$\omega$$         | float | pu   | $$\omega$$ | Rotor speed                          |
+| f                  | float | pu   | f          | Local (nodal) frequency              |
+| $$P_\mathrm{e}$$   | float | pu   | pe         | Measured electrical power generation |
+| $$P_\mathrm{ref}$$ | float | pu   | Pref       | load setpoint                        |
+
+
+### Outputs
+
+<div id="tbl-outputs">
+
+Table 3: Outputs
+</div>
+
+| name             | type  | unit | IEC name | description      |
+| :--------------- | :---- | :--- | :------- | :--------------- |
+| $$P_\mathrm{m}$$ | float | pu   | Pm       | Mechanical power |
+
+
+## Equations & algorithm  
+
+–
+
+## Initial equations / boundary conditions (optional)
+
+The initial values for the system’s states are calculated from the
+initial mechanical power $$P_\mathrm{m\,0}$$, frequency $$f_\mathrm{0}$$ and
+rotation speed $$\omega_\mathrm{0}$$. 
+
+### Helper variables
+
+The following “helper variables” are defined to avoid repetition in the
+definitions of initial states below.
+
+<span id="eq-initReheat">$$
+R_\mathrm{h\,0} = 
+\begin{cases}
+    P_\mathrm{m\,0} ,& \text{if } T_\mathrm{rh} > 0\\
+    0,              & \text{otherwise}
+\end{cases}
+ \qquad(1)$$</span>
+
+<span id="eq-initPc">$$
+P_\mathrm{c\,0} = P_{\text{m}\,0} - \min\left(\max\left(K_{\omega_{\text{cor}}} \cdot 1, \omega_{\text{min}}\right), \omega_{\text{max}}\right)
+ \qquad(2)$$</span>
+
+### Initial states
+
+These initial states refer to the states as defined in
+<a href="#fig-control_diagram" class="quarto-xref">Figure 1</a>.
+
+<span id="eq-initXlp">$$
+x_\mathrm{lp\,0} = P_\mathrm{m\,0}
+ \qquad(3)$$</span>
+
+<span id="eq-initXrh">$$
+x_\mathrm{rh\,0} = R_\mathrm{h\,0}
+ \qquad(4)$$</span>
+
+<span id="eq-initXhp">$$
+x_\mathrm{hp\,0} = P_\mathrm{m\,0}
+ \qquad(5)$$</span>
+
+<span id="eq-initXb">$$
+x_\mathrm{b\,0} = 1
+ \qquad(6)$$</span>
+
+<span id="eq-initXhhp">$$
+x_\mathrm{hhp\,0} = P_\mathrm{m\,0}/x_\mathrm{b}
+ \qquad(7)$$</span>
+
+<span id="eq-initXvip">$$
+x_\mathrm{vlp\,0} = 1
+ \qquad(8)$$</span>
+
+<span id="eq-initXomega">$$
+x_\mathrm{\omega\,0} =  \omega_\mathrm{0}
+ \qquad(9)$$</span>
+
+<span id="eq-initXen">$$
+x_\mathrm{en\,0} = P_\mathrm{c\,0}
+ \qquad(10)$$</span>
+
+<span id="eq-initXip">$$
+x_\mathrm{ip\,0} = P_\mathrm{c\,0}
+ \qquad(11)$$</span>
+
+<span id="eq-initXdp">$$
+x_\mathrm{dp\,0} = 0
+ \qquad(12)$$</span>
+
+<span id="eq-initXp">$$
+x_\mathrm{p\,0} = P_\mathrm{m\,0}
+ \qquad(13)$$</span>
+
+<span id="eq-initXf">$$
+x_\mathrm{f\,0} = f_\mathrm{0}
+ \qquad(14)$$</span>
+
+## Open Questions?
+
+- The island mode control signal does not pass through the electro
```

Comment:
No, they are actually our questions. We are collecting anything that we couldn't clarify, so 
- it is transparent and anyone who reads the document can contribute to the answers (as you did, thanks!)
- readers can see that they are not missing something obvious, but that we had the same question


- first two questions are model developer choice, but we could not find an explanation why they did it like that. 
- yes, I also think it's weird that both modes are active, but this is how it's given in the typical parameters, from the standard.

Reply:
I did some digging. 

- The first question is not really a modeling choise. The output of the island mode does not have to pass through the electrohydraulic transducer because it is an analog/mechanical controller (whereas the output of the PID is an electric signal). I believe this question can be removed.
- The second question is again not a modeling choise. The output of the boiler (and reheater) integrator is pressure that has to be multipled by the valve opening to give the flow/power. Therefore, the multiplication has to be inside the first order transfer function. This question can be also removed.
- Let's keep the last point. It's still not clear for me. I would propose to rephrase as "care should be taken when selecting the island and grid-connected mode parameters".

Reply:
Nice, the first point seems logic to me, I will add that info to the docs. 

The second point still isn't quite satisfying. What I mean is: 

![image](https://github.com/user-attachments/assets/2808fa79-9dd5-4e82-a95d-69901edf2bd2)

Why the multiplication inside the first order element (top), not outside (bottom). 
It changes the lag elemt's behaviour quite a bit, I played around with it a while back.

Reply:
I am not sure what you mean, we could discuss in the next meeting if you like.

I would like to emphasize that in the upper drawing, if you reduce valve opening, e.g. from 1 to 0.5, the output of the first-order will start rising again immediately, because the integrator just raises its output intul the feedback is zero. So you can't half-close the valve properly. 
If you fully close it to zero, then it works, but only then.

Reply:
For the following I am using the notation from neplan: 
![image](https://github.com/user-attachments/assets/46dfcd1b-88ce-41e0-bdd7-41c544566a8d)

This is how I understand it (for simplicity let's ignore the island mode):
- The input to the integrator is v6, which is in steady state (in pu): Poref + Kf *Df 
- The input to the control valve, is v23 (or v22 if not saturated), which is in steady state also (in pu) Poref + Kf *Df
- I understand that the model implements the boiler following mode with feedforward action. The time constant of the boiler should be quite big (in your case 100s, but typically 150-300s), so the pressure changes much more slowly. The fast control of the power is done by adjusting the control valve. However, the expected power demand (which is v6) is also fed forward to the boiler. Hence the boiler and the control valve are coordinated.

Hence, back to your example:
- A command comes to reduce the power e.g. because of frequency increase --> v6 is decreased earlier than v22.
- v6 is fed forward to the boiler integrator, hence the pressure will start dropping slowly with a very small decrease in mechanical power
- Meanwhile the PID controller will reduce v23, hence the control valve will decrease, decreasing the mechanical power faster.
- Eventually, both inputs of the boiler integrator and of the control valve will be the same, hence there will be no "fighting" between the two integrators.

It's trickier to explain it in more detail in writing. If still not clear, to be rediscussed on Friday.

Reply:
I think I could follow, this makes sense. Thanks. It would be great to include that into the docs in a followable way. I will have to see if I have the time